### DISCUSSION TITLE
Fullscreen has find bar

### DISCUSSION THREAD

**Discussion by sheepdoggie** [freeplane/freeplane#1412] [Help]
I did a find while in fullscreen mode and now the find bar remains present whenever I am in fullscreen. How do I get rid of it? Freeplane version: 1.10.3

**Comment by euu2021** [comment]
Right click the background, and disable here:
![image](https://github.com/freeplane/freeplane/assets/77707706/2a3d9f3d-675f-4a98-adb5-085ca56f3fd0)

**Reply by sheepdoggie** [reply]
Ah, yes. I was right clicking the background of the Filter toolbar itself, and that does not work. One must right click the background of the mindmap. Thank you. Seems like any background should work, tho.
